cmd/podman/containers/run.go

```diff
@@ -83,6 +83,9 @@ func runFlags(cmd *cobra.Command) {
 	_ = cmd.RegisterFlagCompletionFunc(gpuFlagName, completion.AutocompleteNone)
 	_ = flags.MarkHidden("gpus")
 
+	noPasswdFlagName := "no-manage-passwd"
```

Comment:
How about --passwd and default to true.
I hate multi dashes in an option and dealing with double negatives.

If I want no passwd management, I enter --passwd=false.

Reply:
I really like that :+1:

Reply:
SGTM

Reply:
Remember also on the remote API to always set the flag to TRUE, unless the user sends a modification.

Reply:
Ok, I think setting it in the spec takes care of that since for remote the spec gets passed over.

Reply:
Sure, I just want to make sure if I call directly from python that the passwd==true is maintained.  IE the entries get added.

Reply:
I see, is there an API endpoint for container run? I am not seeing one. If that is the case then the spec is the only spot we need to worry about right? since remote is the only alternative way to run this command.

Reply:
podman/pkg/api/handlers/libpod/create_container.go